runblender.ipynb

```diff
@@ -31,30 +31,32 @@
     {
       "cell_type": "code",
       "metadata": {
-        "id": "8dFNjYGTgNjR",
-        "colab_type": "code",
-        "colab": {}
+        "id": "8dFNjYGTgNjR"
       },
       "source": [
         "#@title Setup\n",
         "#@markdown Please configure your setup\n",
         "\n",
         "blender_version = 'blender2.91.2' #@param [\"blender2.80\", \"blender2.81\", \"blender2.82\", \"blender2.83\", \"blender2.90.1\", \"blender2.91.2\"] {allow-input: false}\n",
-        "path_to_blend = 'path/to/blend.blend'  #@param {type: \"string\"}\n",
-        "output_path = 'output/dir/in/drive/output####.png'  #@param {type: \"string\"}\n",
+        "#@markdown You can right click a file and click copy path\n",
+        "path_to_blend = '/content/blendfile.blend'  #@param {type: \"string\"}\n",
+        "output_path = '/content/output'  #@param {type: \"string\"}\n",
         "gpu_enabled = True #@param {type:\"boolean\"}\n",
         "cpu_enabled = False #@param {type:\"boolean\"}\n",
-        "#@markdown ---\n"
+        "use_drive = False #@param {type:\"boolean\"}\n",
+        "#@markdown ---\n",
+        "if use_drive:\n",
+        "  from google.colab import drive\n",
+        "  drive.mount('/content/drive')\n",
```

Comment:
I would seperate this still into a seperate code cell. So configuration and execution are distinct

Reply:
Sure